.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+# This action works with pull requests and pushes to branches other than main
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      - main
+
+jobs:
+  app-jest:
+    runs-on: ubuntu-latest
+    name: app-node-${{ matrix.node }}
+    strategy:
+      matrix:
+        node: [16] # One or more Node versions to test against
```

Comment:
Only testing against Node 16 but this allows us to test against multiple versions if we wanted to

Reply:
Nice—very excited to see CI features being added in!